src/condor_credd/condor_credmon_oauth/condor_vault_storer

```diff
@@ -150,10 +167,21 @@ but make sure no other job is using them."
     if [ -n "$BTOKEN" ]; then
         OPTS=("-o" "$BTOKEN-$SERVICE" "${OPTS[@]}")
     fi
-    OPTS=("--vaulttokenttl=28d" "${OPTS[@]}" "--vaulttokeninfile=$VTOKEN-$SERVICE" "--vaulttokenfile=/dev/stdout" "--showbearerurl")
+
+    if $IS_POMS_ANALYSIS; then
+        OPTS=("--vaulttokenttl=28d" "${OPTS[@]}" "--vaulttokeninfile=/dev/stdin" "--vaulttokenfile=/dev/stdout" "--nokerberos"  "--showbearerurl" "$CREDKEY")
```

Comment:
Couldn't this read from a file instead of stdin?  I suggested having a VAULT_TOKEN_FILE environment variable that can override the location of the vault token, or better yet a CONDOR_VAULT_STORER_ID which takes the place of the $ID variable to give a unique name to both the bearer token and the vault token.